### DISCUSSION TITLE
Version 0.2.1 - Add VERSION command

### DISCUSSION THREAD

**Discussion by c3d** [c3d/db48x#64] [Announcements]
Add a `VERSION` command, and git-derived version string in the PGM file

**Full Changelog**: https://github.com/c3d/DB48X-on-DM42/compare/v0.2...v0.2.1

<hr /><em>This discussion was created from the release <a href='https://github.com/c3d/DB48X-on-DM42/releases/tag/v0.2.1'>Version 0.2.1 - Add VERSION command</a>.</em>

**Comment by balta1980** [comment]
Already my calc on it. All good.
